contentcuration/contentcuration/utils/pagination.py

```diff
@@ -79,6 +79,6 @@ def get_order_queryset(request, queryset, field_map):
     if request.GET.get("descending", "true") == "false" and order != "undefined":
         order = "-" + order
 
-    queryset = queryset.order_by() if order == "undefined" else queryset.order_by(order)
+    queryset = queryset.distinct().order_by() if order == "undefined" else queryset.order_by(order)
```

Comment:
Can either do it on another line, or have to do it for both the first and the else querysets.